core/include/edm/cell.hpp

```diff
@@ -64,7 +64,7 @@ struct cell_module {
     channel_id range0[2] = {std::numeric_limits<channel_id>::max(), 0};
     channel_id range1[2] = {std::numeric_limits<channel_id>::max(), 0};
 
-    pixel_segmentation pixel;
+    pixel_segmentation pixel = pixel_segmentation{-8.425, -36.025, 0.05, 0.05};
```

Comment:
I know this is pedantic, but why not
```suggestion
    pixel_segmentation pixel{-8.425, -36.025, 0.05, 0.05};
```

Reply:
makes sense. :p